tools/accuracy_checker/configs/person-reidentification-retail-0300.yml

```diff
@@ -0,0 +1,43 @@
+models:
+  - name: person-reidentification-retail-0300
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/person-reidentification-retail-0300/FP32/person-reidentification-retail-0300.xml
+        weights: intel/person-reidentification-retail-0300/FP32/person-reidentification-retail-0300.bin
+        adapter: reid
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   intel/person-reidentification-retail-0300/FP16/person-reidentification-retail-0300.xml
+        weights: intel/person-reidentification-retail-0300/FP16/person-reidentification-retail-0300.bin
+        adapter: reid
+
+      - framework: dlsdk
+        tags:
+          - FP32-INT8
+        model:   intel/person-reidentification-retail-0300/FP32-INT8/person-reidentification-retail-0300.xml
+        weights: intel/person-reidentification-retail-0300/FP32-INT8/person-reidentification-retail-0300.bin
+        adapter: reid
+
+    datasets:
+      - name: market1501
+
```

Comment:
please pay attention that you read images with opencv (BGR) and then resize them with pillow. Is this preprocessing correct?